<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartAddOHLCSeriesWidget</class>
 <widget class="QWidget" name="DAChartAddOHLCSeriesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add XY Series</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxX">
         <property name="title">
          <string>X</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxT"/>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBoxTAutoincrement">
            <property name="title">
             <string>Autoincrement series</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="labelTInitValue">
               <property name="text">
                <string>Initial value</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditTInitValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelTStep">
               <property name="text">
                <string>Self increasing step size</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditTStepValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxO">
         <property name="title">
          <string>Open</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxO"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxH">
         <property name="title">
          <string>Hight</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxH"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxL">
         <property name="title">
          <string>Low</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxL"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DA::DACollapsibleGroupBox" name="groupBoxC">
         <property name="title">
          <string>Close</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="DA::DADataManagerComboBox" name="comboBoxC"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="DA::DAPySeriesTableView" name="tableViewOHLC"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DADataManagerComboBox</class>
   <extends>QComboBox</extends>
   <header>DADataManagerComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>DA::DACollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header location="global">DACollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAPySeriesTableView</class>
   <extends>QTableView</extends>
   <header>DAPySeriesTableView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
